### PR TITLE
APIGOV-15076 - First PR from Git (Filtering / SSL)

### PR DESCRIPTION
**Pull request by cbordier**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Update TLS/SSL security + filtering sample

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by lbadenhop** [comment]
Hi @alexearnshaw and @andreamussap ,
Can you help me with this PR? Five Checks are failing--I don't know why and I don't know how to fix them.

**Comment by alexearnshaw** [comment]
@lbadenhop The reason the other checks are failing is that the Hugo build is failing on Netlify with this error:
```
11:40:45 PM: Error: Error building site: "/opt/build/repo/content/en/docs/central/connect-api-manager/ssl-tls-advanced.md:9:1": failed to unmarshal YAML: yaml: line 8: mapping values are not allowed in this context
```
You can see any issues with the Netlify build by clicking the Details link next to the failing Netlify check.

File: content/en/docs/central/connect-api-manager/ssl-tls-advanced.md [hunk 1]
```diff
@@ -3,14 +3,20 @@ title: SSL / TLS advanced
...
 draft: false
 weight: 90
-description: By default, the Axway API Manager connection to AMPLIFY Central is
+description: By default, the connection from the agents to AMPLIFY Central is
   SSL secured using TLS1.2 and the appropriate secured default Cipher Suite.
-  However, you can choose to change the default to use different variable values
-  for protocols and supported Cipher Suites that meet your own requirements. See
-  the variable tables in [Deploy your agents] (/docs/central/connect-api-manager/deploy-your-agents/).
+  However, you can choose to change the default behavior and use different variable values
+  for protocols and supported Cipher Suites that meet your own requirements. See details below:
```
Comment (alexearnshaw) [inline content/en/docs/central/connect-api-manager/ssl-tls-advanced.md:9]: @lbadenhop I think the failing check is due to the colon on this line as I think a colon has a special meaning in YAML? In general the YAML description should be a short summary of the entire topic, so I'd suggest moving this text and making it the first paragraph of the topic instead?

Comment (cbordier) [reply content/en/docs/central/connect-api-manager/ssl-tls-advanced.md]: @alexearnshaw, @lbadenhop 
Thanks for letting me know that. I should have seen it on my own as my markdown editor highlighted this line but I did not explain what is wrong.
I removed the : and commit the file.

**Comment by cbordier** [comment]
Hi @lbadenhop 
Issues are fixed. Can you approve and push it?
Cyrille

**Comment by lbadenhop** [comment]
Hi @alexearnshaw - please merge changes (I can't because of the CLA issue), then upload to docs.axway.com. Thanks!

**Pull request merged**